### ISSUE TITLE
Calendar sync cause app crashing

### ISSUE THREAD

**Issue by VonOx** [GladysAssistant/Gladys#23]
Just for your tasklist :wink: (Refer to this topic http://gladysproject.com/forum/viewtopic.php?f=23&t=5939 )

The sync action for Google calendar cause app crash

``` javascript
TypeError: Cannot read property 'dateTime' of undefined
    at saveorUpdateEvent (/home/pi/gladys/api/services/GoogleCalendarService.js:79:16)
    at calendarEventsFound (/home/pi/gladys/api/services/GoogleCalendarService.js:52:10)
    at Request._callback (/home/pi/gladys/node_modules/googleapis/lib/transporters.js:90:7)
    at Request.self.callback (/home/pi/gladys/node_modules/googleapis/node_modules/request/request.js:123:22)
    at Request.emit (events.js:98:17)
    at Request.<anonymous> (/home/pi/gladys/node_modules/googleapis/node_modules/request/request.js:1047:14)
    at Request.emit (events.js:117:20)
    at IncomingMessage.<anonymous> (/home/pi/gladys/node_modules/googleapis/node_modules/request/request.js:998:12)
    at IncomingMessage.emit (events.js:117:20)
    at _stream_readable.js:944:16
    at process._tickDomainCallback (node.js:486:13)
```

**Comment by Pierre-Gilles** [comment]
Thanks :)

**Comment by Pierre-Gilles** [comment]
Fixed by #54 !
